server/models/ParticipantsModel/index.js

```diff
@@ -18,44 +18,31 @@ const ParticipantsModel = {
       .aggregate(allParticipantsQuery(user, queryParams))
       .toArray(),
   allForAssessment: async (db, chart, filtersService) => {
-    const allFiltersDeselected = filtersService.allFiltersInactive()
+    const { filters } = filtersService
 
-    if (allFiltersDeselected) {
+    if (filtersService.allFiltersInactive()) {
       const query = {
         assessment: chart.assessment,
         study: { $in: filtersService.filters.sites, $nin: STUDIES_TO_OMIT },
       }
       return await db
-        .collection(collections.assessmentDayData)
-        .find(query, { projection: ALL_SUBJECTS_MONGO_PROJECTION })
-        .stream()
-    } else {
-      const mongoFacet = filtersService.barChartMongoQueries()
-      const isAggregateQuery = Array.isArray(mongoFacet)
-      const query = isAggregateQuery
-        ? mongoFacet
-        : Object.keys(mongoFacet).reduce((_, next) => {
-            return mongoFacet[next][0].$match
-          }, {})
-
-      const participants = isAggregateQuery
-        ? await AssessmentDayDataModel.index(db, query)
-        : await AssessmentDayDataModel.all(db, query)
-
-      const chartQuery = {
-        assessment: chart.assessment,
-        participant: {
-          $in: isAggregateQuery
-            ? participants[0].participants
-            : participants.map(({ participant }) => participant),
-        },
-      }
-
-      return await db
-        .collection(collections.assessmentDayData)
-        .find(chartQuery, { projection: ALL_SUBJECTS_MONGO_PROJECTION })
-        .stream()
+      .collection(collections.assessmentDayData)
+      .find(query, { projection: ALL_SUBJECTS_MONGO_PROJECTION })
+      .stream()
```

Comment:
nit but this indentation looks off. Surprised prettier didn't catch it.

Reply:
[formatting fixes](https://github.com/AMP-SCZ/dpdash/pull/659/commits/ad8bf5c42ecf247244eda6a06081123d225c4863)